### DISCUSSION TITLE
"Observer" function is only for enterprise or also comunity version has it?

### DISCUSSION THREAD

**Discussion by tarucan** [gravitational/teleport#37783] [Q&A] [answered]
Hello, as title says.
I tried to edit auditor rules by webgui ad admin, i add:
join_sessions:
      - name: Join prod sessions
        roles : ['prod-access']
        kinds: ['k8s', 'ssh']
        modes: ['moderator', 'observer']

Saved and when i reopen i saw this:
    join_sessions:
    - kinds:
      - ssh
      modes:
      - observer
      name: Join prod sessions
      roles:
      - prod-access

totally changed, and i can not Join session as observer.
My question is, i did something wrong or this function is only for paid version?
Thanks.

**Comment by stevenGravy** [comment, marked as answer]
Correct, only enterprise supports observer and moderator. You should be able to join sessions.

**Reply by colinmollenhour** [reply]
I'm currently on Teleport Community 15.4.4 and I can't seem to do observer or peer either - is this a role issue or none of the three modes are supported for Community?
![image](https://github.com/gravitational/teleport/assets/38738/3c23e82f-7727-49c7-800a-5af046544982)

EDIT: It would be really great if the docs clearly stated what features were Community and what were Enterprise...

**Reply by colinmollenhour** [reply]
I don't know how or why, but I seemed to have fixed it.. Putting `join_sessions` in a role by itself doesn't seem to work, but it does work when I added to the role that gives node access.. Doesn't make sense to me but, ok.

**Reply by webvictim** [reply]
We do have documentation on feature comparisons between Community and Enterprise editions: https://goteleport.com/docs/choose-an-edition/introduction/#comparing-editions

There is also a note in the moderated sessions documentation stating that Teleport Enterprise is required: https://goteleport.com/docs/access-controls/guides/moderated-sessions/

**Reply by colinmollenhour** [reply]
The feature comparison is not very precise, no direct mention of Observer or Peer modes. It is clear that Enterprise is required for Moderated Sessions, but only by omission of not being mentioned specifically as Enterprise can one deduce that Peer and Observer are not Enterprise features. I thought they were Community but when it didn't work and I couldn't find a concrete mention of it, I began to doubt and had to really dig. I'm not the only one as evidenced by this issue.

Just a suggestion.. GitLab does a really good job of documenting the capabilities of different editions. 
![image](https://github.com/gravitational/teleport/assets/38738/5b3ba52f-1a71-4002-b791-dba34d41d663)
